File: Module_1/Exercises_Python/01_divisor.py
num_A = int(input("Digite um número: "))
num_B = int(input("Digite outro número: "))

if num_A > num_B:
    if num_A % num_B == 0:
        print("O valor de A é divisível por B!")
    else:
        print("O valor de A não é divisível por B.")

else:
    print("O valor de A é menor que o valor de B!")

File: Module_1/Exercises_C/01_divisor.c
#include <stdio.h>
#include <stdlib.h>

int main(){

    int numA, numB;

    printf("Digite um valor para A: ");
    scanf("%d", &numA);

    printf("Digite um valor para B: ");
    scanf("%d", &numB);

    if(numA > numB){
        if(numA % numB == 0){
            printf("\nO valor de A é divisível por B!\n");
        } else {
            printf("O valor de A não é divisível or B.\n");
        }
    } else {
        printf("A não é maior que B!\n");
    }

   return 0;
}



